pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -1830,6 +1866,10 @@ type BackupStatus struct {
 	BackupSize int64 `json:"backupSize,omitempty"`
 	// CommitTs is the snapshot time point of tidb cluster.
 	CommitTs string `json:"commitTs,omitempty"`
+	// StartTs is the log backup start ts.
```

Comment:
What's the difference with `spec.StartTs`?

Reply:
`StartTs` is the log start ts, `CommitTs` is the snapshot time point, they are similar, both the time when backup takes effect, i think we use one of them is ok. i will fix it.